iidm/iidm-api/src/main/java/com/powsybl/iidm/network/Network.java

```diff
@@ -70,6 +70,13 @@
          * Get the switch count.
          */
         int getSwitchCount();
+
+        /**
+         * Get a Bus.
+         */
+        default Bus getBus(String id) {
+            return null;
```

Comment:
It would be better to throw an exception.

Reply:
@mathbagu didn't agree with that ! For me throwing an exception is more logic,
We should at least warn the user for using unoverridden function.
what do you think @mathbagu ?

Reply:
> @mathbagu didn't agree with that ! For me throwing an exception is more logic,
> We should at least warn the user for using unoverridden function.
> what do you think @mathbagu ?

It seems to me that throwing an exception is what we have always done in such a case and seems more safe to me.